recipes/scikit-optimize/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "scikit-optimize" %}
+{% set version = "0.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "d8934c33e5958227d2ff97a4fae2134d2a00432b99e7f9d90b17e45cb1da9bb5" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - numpy
+    - scipy
+    - scikit-learn >=0.18
+    - matplotlib
+
+test:
+  imports:
+    - skopt
+    - skopt.learning
+    - skopt.learning.gaussian_process
+    - skopt.optimizer
+    - skopt.space
+
+about:
+  home: https://scikit-optimize.github.io/
+  license_file: {{ RECIPE_DIR }}/LICENSE.md
```

Comment:
I personally review 3-5 recipes per day and rarely leave any sitting ready to merge for more than 6 hours (and often I get to them within the hour). I've reviewed 169 closed PRs and currently have 21 open. Of the 21, 16 are failing their builds, and at least 2 are waiting for important requested changes even though they are passing.  

My impression is the backlog is largely abandoned recipes or recipes in which there is limited reviewing expertise within the community (e.g. R packages or packages with tricky non-python builds), not because of reviewers are holding up merges trying to uphold community standards.